routers/api/v1/repo/issue_tracked_time.go

```diff
@@ -212,6 +214,18 @@ func ListTrackedTimesByRepository(ctx *context.APIContext) {
 	//   description: name of the repo
 	//   type: string
 	//   required: true
+	// - name: user
+	//   in: query
+	//   description: optional filter by user
+	//   type: string
+	// - name: since
+	//   in: query
+	//   description: Only show times updated after the given time. This is a timestamp in ISO 8601 format
+	//   type: strfmt date-time
+	// - name: before
+	//   in: query
+	//   description: Only show times updated before the given time. This is a timestamp in ISO 8601 format
+	//   type: strfmt date-time
```

Comment:
```suggestion
	//   type: string
	//   format: date-time
```

Reply:
@zeripath i changed it already to `string date-time`

should i change it to string?

Reply:
Well it won't be a valid swagger otherwise and it won't pass CI